03LessonThree/homework/git-scm.html

```diff
@@ -52,9 +53,9 @@
           </aside>
         </div>
     </main>
-  </footer>
+  <footer>
         This open sourced site is hosted on GitHub.<br/> Patches, suggestions and comments are welcome.
-    </div>
+    </footer>
 </body>
 
 </html>

```

Comment:
Sounds good Justin thank you sr.

On Thu, Aug 4, 2016 at 8:18 AM, Justin Lugo <notifications@github.com>
wrote:

> In 03LessonThree/homework/git-scm.html
> <https://github.com/AustinCodingAcademy/intro-workbook/pull/51#discussion_r73519568>
> :
>
> >          This open sourced site is hosted on GitHub.<br/> Patches, suggestions and comments are welcome.
> > -    </div>
> > +    </footer>
> >  </body>
> >
> >  </html>
>
> You closed all of the remaining semantic tags. Make sure you focus on
> indentation to improve code readability on the next assignment.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/AustinCodingAcademy/intro-workbook/pull/51/files/fa527cce973809b6e38830323da2dcb8a9e574ee..da3bee6d10280b52e8deffeb57cdd644fecad2a7#r73519568>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATT6krScYA7ONTxwXBx9eZfPqX2rGlQTks5qceaVgaJpZM4JQVyZ>
> .
>